spec/factories/protocol.rb

```diff
@@ -133,9 +139,13 @@
     factory :protocol_federally_funded, traits: [:funded, :federal]
     factory :protocol_with_sub_service_request_in_cwf, traits: [:with_sub_service_request_in_cwf, :funded, :federal]
     factory :study_with_blank_dates, traits: [:study, :pending, :blank_funding_start_dates, :blank_start_and_end_dates]
+    factory :protocol_invalid_study_with_questions, traits: [:without_validations, :study, :with_study_type_questions_group]
```

Comment:
See previous comment for naming convention. What makes this invalid? Should that be without_validations?